db/migrate/add_attachment_image_to_admin/20160929005417_abouts.rb

```diff
@@ -0,0 +1,11 @@
+#class AddAttachmentImageToAdmin::Abouts < ActiveRecord::Migration
```

Comment:
Also, delete this folder `add_attachment_image_to_admin/20160929005417_abouts.rb`